File: src/main/java/com/teamwizardry/wizardry/common/spell/component/EffectChain.java
package com.teamwizardry.wizardry.common.spell.component;

public class EffectChain extends SpellChain
{
    public EffectChain(ModuleEffect effect)
    {
        super(effect);
    }
    
    @Override
    public EffectInstance toInstance(Interactor caster)
    {
        return (EffectInstance) super.toInstance(caster);
    }
}

File: src/main/java/com/teamwizardry/wizardry/common/spell/component/EffectChain.kt
package com.teamwizardry.wizardry.common.spell.component

class EffectChain(effect: ModuleEffect?) : SpellChain(effect) {
    override fun toInstance(caster: Interactor): EffectInstance {
        return super.toInstance(caster) as EffectInstance
    }
}